cmd/fetch_repo/module.go

```diff
@@ -61,21 +60,19 @@ func fetchModule(dest, importpath, version, sum string) error {
 		return fmt.Errorf("error closing temporary go.mod: %v", err)
 	}
 
-	buf := &bytes.Buffer{}
 	cmd := exec.Command(goPath, "mod", "download", "-json", importpath+"@"+version)
-	cmd.Stdout = buf
-	cmd.Stderr = os.Stderr
-	dlErr := cmd.Run()
+	out, dlErr := cmd.Output()
```

Comment:
`cmd.Output` may truncate stderr. Instead, let's set `cmd.Stderr` to a separate `*bytes.Buffer`.

Reply:
Thanks. Done. Happy to follow any styling, variable renaming suggestions, etc. as well.